src/files2.js

```diff
@@ -185,6 +185,7 @@ FOAM_FILES([
   { name: "foam/dao/ArraySink" },
   { name: "foam/dao/DAOInterceptor" },
   { name: "foam/dao/AbstractDAO" },
+  { name: "foam/dao/CopyFromDAO" },
```

Comment:
Question: Is this necessary? I only added it because foam.dao.DAO is listed in both.

Reply:
I don't know what files2.js is.